src/coordination/raft_state.cpp

```diff
@@ -162,6 +165,28 @@ auto RaftState::AddCoordinatorInstance(coordination::CoordinatorToCoordinatorCon
   }
 }
 
+auto RaftState::GetCoordinatorInstances() const -> std::vector<CoordinatorToCoordinatorConfig> {
+  std::vector<ptr<srv_config>> srv_configs;
+  raft_server_->get_srv_config_all(srv_configs);
+
+  auto aux1 = raft_server_->get_aux(0);
+  auto aux2 = raft_server_->get_aux(2);
+  auto aux3 = raft_server_->get_aux(2);
```

Comment:
Why get_aux(2) two times?

Reply:
Seems like this was testing, so I think we should remove this, right?

Reply:
yes